qemu/tests/cfg/fio_windows.cfg

```diff
@@ -1,16 +1,31 @@
 - fio_windows:
     type = fio_windows
     only Windows
+    images += " stg"
+    image_size_stg = 10G
```

Comment:
win2008 disk formate may take very long time,are you going to run fio on formated disk?
If yes,just make sure the format process will not timeout.

Reply:
agree with lijinlijin.  
set image_size_stg = 1G, is enough?

Reply:
Hi lijin and xueqiang,

Thanks for your advise, change the image_size_stg = 2G, there also has a log file save to here,  leave some disk space.
And try to run on win2008-32, it can be passed smoothly.
 (1/2) smp_16.4096m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.Win2008.i386.sp2.io-github-autotest-qemu.fio_windows.with_system_disk: PASS (375.76 s)
 (2/2) smp_16.4096m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.Win2008.i386.sp2.io-github-autotest-qemu.fio_windows.with_data_disk: PASS (366.85 s)